frontend/src/services/datauri/datauri.service.ts

```diff
@@ -0,0 +1,11 @@
+class Datauri {
```

Comment:
just create a helper `getDataUrl` and move it to the `frontend/src/helpers/file`